src/components/ContextMenu.scss

```diff
@@ -73,4 +73,19 @@
   .context-menu-option:focus {
     z-index: 1;
   }
+
+  @media #{$media-query} {
+    .context-menu-option {
+      display: block;
+      margin-inline-end: 0;
```

Comment:
I think that line-80 can be removed - or ?

Reply:
```suggestion
```